tests/gui/test_suggestions.py

```diff
@@ -5,7 +5,12 @@
 def test_velocity_suggestions(centered_pair_predictions):
```

Comment:
Add some tests in this file for each of the "[method functions](https://github.com/talmolab/sleap/blob/2962d6480bb796592e78e0b031594042639d3985/sleap/gui/suggestions.py#L58)" to check that only suggest on the videos we specify.